xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-ui/src/XWiki/Lightbox/WebHome.xml

```diff
@@ -0,0 +1,838 @@
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </scope>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <content>{{velocity}}
+#set ($lightboxConfigDoc = $xwiki.getDocument('XWiki.Lightbox.LightboxConfiguration'))
+#set ($lightboxConfigObj = $lightboxConfigDoc.getObject('XWiki.Lightbox.LightboxConfigurationClass'))
+#if ($lightboxConfigObj.getValue('isLightboxEnabled').equals(1))
+  #set ($discard = $xwiki.linkx.use($services.webjars.url('org.xwiki.contrib:webjar-blueimp-gallery', 'css/blueimp-gallery.min.css'), {'type': 'text/css', 'rel': 'stylesheet'}))
+  #set ($discard = $xwiki.jsx.use('XWiki.Lightbox.WebHome'))
+  #set ($discard = $xwiki.ssx.use('XWiki.Lightbox.WebHome'))
+  {{html clean="false"}}
+  &lt;!-- The Gallery lightbox dialog. -->
+  &lt;div
+    id="blueimp-gallery"
+    class="blueimp-gallery"
+    aria-label="image gallery"
+    aria-modal="true"
+    role="dialog"
+  >
+    &lt;div class="slides" aria-live="polite">&lt;/div>
+    &lt;h3 class="title">&lt;/h3>
+    &lt;a
+      class="prev" aria-controls="blueimp-gallery"
+      aria-label="previous slide" aria-keyshortcuts="ArrowLeft"
+    >&lt;/a>
+    &lt;a
+      class="next" aria-controls="blueimp-gallery"
+      aria-label="next slide" aria-keyshortcuts="ArrowRight"
+    >&lt;/a>
+    &lt;div class="controls btn-group">
+      &lt;a
+        class="btn fullscreen" id="lightboxFullscreen" aria-controls="blueimp-gallery"
+        aria-label="fullscreen" aria-keyshortcuts="Fullscreen"
+      >$services.icon.renderHTML('arrows')&lt;/a>
+      &lt;a
+        class="btn download" id="lightboxDownload" aria-controls="blueimp-gallery"
+        aria-label="download" aria-keyshortcuts="Download"
+      >$services.icon.renderHTML('download')&lt;/a>
+      &lt;a
+        class="btn autoPlay" aria-controls="blueimp-gallery" aria-label="autoPlay slideshow"
+        aria-keyshortcuts="Space" aria-pressed="false"
+      >$services.icon.renderHTML('play')&lt;/a>
+      &lt;a
+        class="btn escape" aria-controls="blueimp-gallery"
+        aria-label="escape" aria-keyshortcuts="Escape"
+      >$services.icon.renderHTML('cross')&lt;/a>
+    &lt;/div>
+    &lt;div class="lightboxDescription">
+      &lt;div class="descriptionCol">
+        &lt;span class="title">&lt;/span>
+      &lt;/div>
+      &lt;div class="descriptionCol">
+        &lt;span class="publisher">&lt;/span>
+        &lt;span class="date">&lt;/span>
+       &lt;/div>
+    &lt;/div>
+    &lt;ol class="indicator">&lt;/ol>
+  &lt;/div>
+  {{/html}}
+#end
+{{/velocity}}</content>
+    </property>
+    <property>
+      <extensionPointId>org.xwiki.platform.template.header.after</extensionPointId>
+    </property>
+    <property>
+      <name>org.xwiki.platform.template.header.after.image.lightbox</name>
```

Comment:
Should this be `org.xwiki.platform.image.after` or `org.xwiki.platform.image.header.after.lightbox` (or variations thereof) per the [naming conventions](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HUserInterfaceExtensionPointNaming), i.e., use the `moduleName` of the module declaring the UIX, not of the module declaring the UIXP.

Reply:
I'm not happy with the naming conventions we have for UIX. I commented on https://forum.xwiki.org/t/naming-convention-for-user-interface-extension-points/6416/27?u=mflorea . I agree that the prefix should be the group id and module that provides the UIX (not the UIXP). So in this case ``org.xwiki.platform.image.*``. As for the suffix, I'd choose from:

* org.xwiki.platform.image.lightbox
* org.xwiki.platform.image.lightboxDialog
* org.xwiki.platform.image.lightboxTemplate

So I'd put the focus on what this UIX does / provides rather than on what UIXP is contributes to.